app/get_proc_arg.go

```diff
@@ -60,7 +60,7 @@ const (
         if [ ! -f $proc/cmdline ]; then
           continue
         fi
-        if [[ $(cat $proc/cmdline | tr '\000' '\n' | sed -n '1p') == "k3s" ]]; then
+        if [[ $(cat $proc/cmdline | tr '\000' '\n' | sed -n '1p' | tr '/' '\n' | tail -n1) == "k3s" ]]; then
```

Comment:
Can we use the exact same syntax as we detecting `kubelet`? Or even better, reuse the function so this error will not happen again.

Also noticed that a few lines above, the error message as `No arg root-dir` instead of `No arg %s`.

Maybe it's better for the script to return the line we're interested in (the cmdline of `kubelet` or `k3s`), then let go code process it. Since in k3s case, you will need to find two parameters, which make it harder to generalize the function.